mobile/Screen/MapScreen.js

```diff
@@ -36,6 +36,26 @@ const styles = StyleSheet.create({
 });
 
```

Comment:
Were you able to test the changes on this screen out? I saw you commented out the MapScreen altogether before